controllers/clustergroupupgrade_controller.go

```diff
@@ -663,6 +671,11 @@ func (r *ClusterGroupUpgradeReconciler) doManagedPoliciesExist(
 	clusterGroupUpgrade.Status.ManagedPoliciesContent = make(map[string]string)
 
 	for _, managedPolicyName := range clusterGroupUpgrade.Spec.ManagedPolicies {
+		if policyEnforce[managedPolicyName] {
+			r.Log.Info("Ignoring policy " + managedPolicyName + " with remediationAction enforce")
```

Comment:
```suggestion
			r.Log.Info("[doManagedPoliciesExist] Ignoring policy " + managedPolicyName + " with remediationAction enforce")
```
to keep it consistent with the rest of the file.